server/a_socials/settings/prod.py

```diff
@@ -13,7 +13,7 @@
     'default': dj_database_url.config(default=dotenv.get('DATABASE_URL'))
 }
 
-db_from_env = dj_database_url.config(conn_max_age=500)
+db_from_env = dj_database_url.config(conn_max_age=None)
```

Comment:
Prod settings are not used in the test if I recall correctly.

Reply:
Thanks at @andela-sjames the work on this PR is not complete and we're trying to figure out why the tests are failing. 
This is helpful feedback